ex03.ex

```diff
@@ -77,7 +89,15 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains([],a) do
```

Comment:
# 5

This is a close call, but I think that this is better as

~~~ elixir
def list_contains([], _value), do: false
def list_contains([h|t], h),   do: true
def list_contains([_h|t]),     do: list_contains(t, value)
~~~

My thinking is that the structure is clearer—the three function heads are parallel, whereas in the version with an if the decision is made in the head in the first line and the body in the second.

As I said, this is something of a fine point, but you're clearly getting all this, so I'm going to try to push you a little more.